packages/cluster-ui/src/barCharts/barCharts.tsx

```diff
@@ -38,26 +34,71 @@ const latencyBars = [
   ),
 ];
 
+const maxMemUsageBars = [
+  bar("max-mem-usage", (d: StatementStatistics) => d.stats.max_mem_usage?.mean),
+];
+
+const networkBytesBars = [
+  bar("network-bytes", (d: StatementStatistics) => d.stats.bytes_sent_over_network?.mean),
+]
+
+const retryBars = [
+  bar(
+    "count-retry",
+    (d: StatementStatistics) =>
+      longToInt(d.stats.count) - longToInt(d.stats.first_attempt_count),
+  ),
+];
+
+const rowsReadStdDev = bar(cx("rows-read-dev"), (d: StatementStatistics) =>
+  stdDevLong(d.stats.rows_read, d.stats.stat_collection_count),
+);
+const bytesReadStdDev = bar(cx("rows-read-dev"), (d: StatementStatistics) =>
+  stdDevLong(d.stats.bytes_read, d.stats.stat_collection_count),
+);
 const latencyStdDev = bar(
   cx("bar-chart__overall-dev"),
   (d: StatementStatistics) => stdDevLong(d.stats.service_lat, d.stats.count),
 );
-const rowsStdDev = bar(cx("rows-dev"), (d: StatementStatistics) =>
-  stdDevLong(d.stats.num_rows, d.stats.count),
+const maxMemUsageStdDev = bar(cx("max-mem-usage-dev"), (d: StatementStatistics) =>
+  stdDevLong(d.stats.max_mem_usage, d.stats.stat_collection_count),
+)
+const networkBytesStdDev = bar(cx("network-bytes-dev"), (d: StatementStatistics) =>
+  stdDevLong(d.stats.bytes_sent_over_network, d.stats.stat_collection_count),
 );
 
 export const countBarChart = barChartFactory("grey", countBars, approximify);
-export const retryBarChart = barChartFactory("red", retryBars, approximify);
-export const rowsBarChart = barChartFactory(
+export const rowsReadBarChart = barChartFactory(
   "grey",
-  rowsBars,
+  rowsReadBars,
   approximify,
-  rowsStdDev,
+  rowsReadStdDev,
   formatTwoPlaces,
 );
+export const bytesReadBarChart = barChartFactory(
+  "grey",
+  bytesReadBars,
+  Bytes,
+  bytesReadStdDev,
+  // TODO(asubiotto): What's a legend formatter, do we need it?
```

Comment:
@asubiotto Legend formatter can convert the numeric stat values to strings in the pop up legend using a *different* method than the "primary" formatter (`Bytes`) here for instance. I think in your case you want them to be consistent so you can omit the argument.

Here's an example where I added lots of numbers after the decimal point and just used `toString` as the legend formatter, while the primary formatter remained `approximify`.
![Screen Shot 2021-01-26 at 1 26 08 PM](https://user-images.githubusercontent.com/986307/105887695-0ff29f00-5fda-11eb-9781-139643afa78a.png)